testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -426,6 +426,31 @@ def forward(self, x):
 output = np.mean(input, axis=(3, 4), keepdims=True)
 save_onnx_data_and_model(input, output, 'reduce_mean3d', 'ReduceMean', axes=(3, 4), keepdims=True)
 
+class Split(nn.Module):
+
+    def __init__(self, *args, **kwargs):
+        super(Split, self).__init__()
+        self.split_size_sections = \
+            kwargs.get('split_size_sections', 1)
+        self.dim = kwargs.get('dim', 0)
+
+    def forward(self, x):
+        tup = torch.split(x, self.split_size_sections, self.dim)
+        return torch.cat(tup)
```

Comment:
I have used this because, usually, `torch.split` returns a python `tuple` which if converted to a tensor here will render the code useless as the layer will be saved as `Constant` and not `Split`.
And we cannot save python tuple in `npy` file as they are not produced correctly in the `cpp` tests and moreover they will require a lot of changes across some files to make tuples work in the code. Hence, this trick tests the `Split` layer with minimal changes in the code. Though I am open to suggestions on improving this.

Reply:
I have run the tests locally they, pass on my system.